src/test/java/step2/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+package step2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+
+class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
```

Comment:
```suggestion
    @NullAndEmptySource
    @ParameterizedTest(name = "입력값: {0}")
    public void splitAndSum_null_또는_빈문자(String input) {
        int result = StringAddCalculator.splitAndSum(input);
        assertThat(result).isEqualTo(0);
    }
```
NullAndEmptySource 이런 어노테이션도 있으니 ParameterizedTest와 함께 테스트 해보세요 👏